### PR TITLE
Update local SAR to scope to a namespace correctly

### PR DESCRIPTION
**Pull request by smarterclayton**
SAR checks on namespaces requires the namespace and resourceName fields
to be set.

### CODE REVIEW

**Comment by smarterclayton** [comment]
@zgalor

**Comment by smarterclayton** [comment]
As per #15687

**Comment by csrwng** [comment]
The sar looks good to me, it wouldn't hurt for @deads2k to take a look though.
@deads2k the sar argument is submitted by the oauth proxy image back to the api server: 
https://github.com/openshift/oauth-proxy/blob/master/providers/openshift/provider.go#L392

File: contrib/prometheus/prometheus.yml [hunk 1]
```diff
@@ -1,28 +0,0 @@
-# my global config
```
Comment (deads2k) [inline contrib/prometheus/prometheus.yml]: why is this changing?

**Comment by deads2k** [comment]
lgtm

File: contrib/prometheus/prometheus.yml (hunk 1 above)
Comment (smarterclayton) [inline contrib/prometheus/prometheus.yml]: Forgot it existed, removing.  This was very old and is useless now.

**Pull request merged**